common/BasicMultiQueueApplication.hpp

```diff
@@ -127,6 +127,12 @@ class BasicMultiQueueApplication : public virtual MonoDeviceApplication
 			// since we requested a device that has a compute capable queue family (unless `getQueueRequirements` got overriden) we're sure we'll get at least one family capable of compute
 			assert(m_computeQueue.famIx!=QueueAllocator::InvalidIndex);
 
+			// First thing to make sure we have is a compute queue (so nothing else fails allocation) which should be able to do image transfers of any granularity (transfer only queue families can have problems with that)
```

Comment:
wait for me to merge https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/82

and overwrite all this